src/main/java/zd/zero/waifu/motivator/plugin/motivation/MotivationFactory.kt

```diff
@@ -98,18 +101,20 @@ object MotivationFactory {
             val project = motivationEvent.project
             assetSupplier()
                 .doOrElse({ asset ->
-                    val motivation =
-                        motivationConstructor(
-                            asset
-                        ).setListener { lifecycleListener.onDispose() }
+                    if (::alertMotivation.isInitialized) {
+                        alertMotivation.closeNotification()
```

Comment:
If this was intended to lay the ground work for #225 , what we will end up with is a bunch of notifications opening and closing before the final notification shows, just as a FYI.

Reply:
Hmmm, for this scenario, I plan to have some alerts to get debounced, not all alerts need to be debounced so I plan to only add the debouncer on the specific type of alerts. Do you have a better idea to tackle this problem?

Reply:
I think whatever the solution is, the user should be presented with the single most relevant notification at the time.